lib/ot-date.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+module.exports = function (moment) {
+    /**
+     * Open Token Date Class to provide simple date functionality
+     * using Moment.js.
+     *
+     * @param {Moment} m
+     */
+    class OtDate {
+        constructor(m) {
+            if (! (m instanceof moment)) {
+                throw new Error("Use helper functions instead");
+            }
+
+            this.moment = m.utc();
+        }
+
+
+        /**
+         * Checks if the set time is before the time passed in.
+         *
+         * @param {Date} otherDate
+         * @return {boolean} whether the date is before
+         */
+        isBefore(otherDate) {
+            return this.moment.isBefore(otherDate);
+        }
+
+
+        /**
+         * Adds time to the set time.
+         * You can send in multiple parameters to move the date up.
```

Comment:
"up" can be used in two different ways.

Closer:
> Oh crap!  They moved the project due date up on us again!

Farther:
> Let's move the project due date up beyond the freeze window.

Could you change the wording?